Dockerfile

```
@@ -29,6 +27,6 @@ WORKDIR /home/tidepool
 
 USER tidepool
 
-COPY --from=development --chown=tidepool /go/src/github.com/tidepool-org/shoreline/dist/shoreline .
+COPY --from=development --chown=tidepool /go/src/github.com/mdblp/shoreline/dist/shoreline .
```

As a consequence of previous change we should copy it from /go/src/github.com/tidepool-org
```suggestion
COPY --from=development --chown=tidepool /go/src/github.com/tidepool-org/shoreline/dist/shoreline .
```